libs/vkd3d/command.c

```diff
@@ -8306,15 +8302,14 @@ static void d3d12_command_list_set_root_descriptor_va(struct d3d12_command_list
 }
 
 static void d3d12_command_list_set_root_descriptor(struct d3d12_command_list *list,
-        VkPipelineBindPoint bind_point, unsigned int index, D3D12_GPU_VIRTUAL_ADDRESS gpu_address)
+        struct vkd3d_pipeline_bindings *bindings, unsigned int index, D3D12_GPU_VIRTUAL_ADDRESS gpu_address)
```

Comment:
There are some warnings around enums still be used, and end up translating to NULL pointer :scream:

Reply:
I'm not seeing any; what's triggering those?